<?xml version="1.0"?>
<launch>
    <node pkg="mcr_perception_monitors" type="blob_tracking_error_monitor_node" name="blob_tracking_error_monitor" ns="mcr_perception" output="screen" respawn="true">
    	<remap from="~pose_error" to="/component_wise_pose_error_calculator/pose_error"/>
    </node>
</launch>